packages/server/lib/makeDataContext.ts

```diff
@@ -62,9 +62,6 @@ export function makeDataContext (options: MakeDataContextOptions): DataContext {
       validateConfig: configUtils.validate,
       updateWithPluginValues: config.updateWithPluginValues,
       setupFullConfigWithDefaults: config.setupFullConfigWithDefaults,
-      validateRootConfigBreakingChanges: configUtils.validateNoBreakingConfigRoot,
```

Comment:
Can we remove everything passed in from `@package/config` and directly consume it in the DataContext package?

Also, is there any need to keep the config validation logic in the server or can it all be moved to the config package and consume it from the Data Context page so all config logic is in one place?

Reply:
Logged a future issue here: https://cypress-io.atlassian.net/browse/UNIFY-1431